src/ui/layout/navbar.svelte

```diff
@@ -1,31 +1,38 @@
 <script lang="ts">
   import { Theme } from "../theme/theme";
-  import NavBarHeader from "./navbar-header.svelte";
-  import SectionLayout from "./section-layout.svelte";
   import type { BrandingAppearance } from "../../entities/branding";
+  import type { Snippet } from "svelte";
 
-  export let brandingAppearance: BrandingAppearance | null | undefined =
-    undefined;
-  $: style = new Theme(brandingAppearance).productInfoStyleVars;
+  type Props = {
+    headerContent?: Snippet<[]>;
+    bodyContent?: Snippet<[]>;
+    brandingAppearance?: BrandingAppearance | null;
+  };
 
-  export let headerContent;
-  export let bodyContent: () => any;
+  const {
+    headerContent,
+    bodyContent,
+    brandingAppearance = undefined,
+  }: Props = $props();
 
-  export let showCloseButton: boolean;
-  export let onClose: (() => void) | undefined = undefined;
+  const style = $derived(new Theme(brandingAppearance).productInfoStyleVars);
 </script>
 
 <div class="rcb-ui-navbar" {style}>
-  <SectionLayout layoutStyle="justify-content: flex-end;">
-    {#snippet header()}
-      <NavBarHeader {showCloseButton} {onClose}>
-        {@render headerContent?.()}
-      </NavBarHeader>
-    {/snippet}
-    {#snippet body()}
-      {@render bodyContent?.()}
-    {/snippet}
-  </SectionLayout>
+  <div class="layout-wrapper-outer" style="justify-content: flex-end;">
```

Comment:
What limitation did you find in `SectionLayout` to remove it and duplicate these divs in both navbar and main?

-- Not opposed, just curious

Reply:
Yeah I'm not super proud of this since I added some style duplication with the wrappers and so on. The reason is that the new navigational bar has different paddings the left and the right because of the back/close buttons. Although now that I did a second iteration, and we got rid of the chevron back button in the mobile view it maybe would be a bit easier to keep it.